go.mod

```diff
@@ -37,6 +41,7 @@ require (
 )
 
 replace (
+	gopkg.in/gcfg.v1 => github.com/mandelsoft/gcfg v1.2.4-0.20191118133708-71f203a139d3
```

Comment:
Still not a fan of using a personal forked repo here. If it's absolutely necessary can we copy it over to `third_party/forked/gcfg`? See https://github.com/kubernetes/kubernetes/tree/master/third_party/forked for an example.

Reply:
I agree with this. Really not a fan of this as well.

Reply:
we replaced the tags section by a tag attribute containing a JSON map.